api/proto/info/info.proto

```diff
@@ -278,6 +278,20 @@ message ZCellularModuleInfo {
     string firmware_version = 3;
 }
 
+// Device Run State
+enum ZDeviceRunState {
+    // Just to handle Un-Initialized runState variable Internally
+    ZDEVICE_RUN_STATE_UNSPECIFIED = 0;
+    ZDEVICE_RUN_STATE_ONLINE = 1;
+    // Device Reboot in Progress. During bootup, Device reports "BOOTING"
```

Comment:
I think this comment should say it is on its way down as part of rebooting, and that BOOTING is when it is on the way up.
Also, for REBOOTING, BOOTING, and BASEOS_UPGRADING we should document that the device is partially operational.

Reply:
Changed the description to:

// Device Reboot in Progress. Device is in the process of going down.
// Once the device comes back up and Eve is started, device reports the
// the state as "BOOTING"

Am thinking we should just leave at this. Would introducing the phrase "Partially Operational" is more confusing? It opens up a can of worms??